module/VuFindTheme/tests/unit-tests/src/VuFindTest/ThemeInfoTest.php

```diff
@@ -213,6 +213,47 @@ public function testFindInThemes()
         );
     }
 
+    /**
+     * Test getMergedConfig()
+     *
+     * @return void
+     */
+    public function testGetMergedConfig()
+    {
+        // Parent
+        $ti = $this->getThemeInfo();
+        $parentJS = $ti->getMergedConfig('js');
+        $this->assertEquals(['hello.js'], $parentJS);
+        // recursive
+        $parentHelpers = $ti->getMergedConfig('helpers');
+        $this->assertEquals(
+            'fooFactory',
+            $parentHelpers['factories']['foo']
+        );
+
+        // Child with parents merged in
+        $ti->setTheme('child');
+        $childJS = $ti->getMergedConfig('js');
+        $this->assertEquals(['extra.js', 'hello.js'], $childJS);
+        // recursive
+        $childHelpers = $ti->getMergedConfig('helpers');
+        $this->assertEquals(
+            ['fooOverrideFactory', 'fooFactory'],
+            $childHelpers['factories']['foo']
+        );
```

Comment:
It made sense to me that if we're merging values (like with JS and CSS) that the child ones should come before the parent ones. Is there a reason why we should switch this?

Reply:
I would normally expect the parent values to come before the child values, since the child values are often adding to the parent values, and may depend upon the things that precede them. However, I'm not sure if this expectation is relevant to what we're doing here, since I think in the context of icons, we're either overriding specific values or else adding things into a list where order is unimportant. I'm just speaking to what I would expect in terms of CSS/JS behavior, which may not really apply to this.